crypto/kyber/p_kyber_test.cc

```diff
@@ -141,21 +141,21 @@ TEST(Kyber512Test, KeySize) {
   EXPECT_TRUE(EVP_PKEY_keygen_init(kyber_pkey_ctx));
   EXPECT_TRUE(EVP_PKEY_keygen(kyber_pkey_ctx, &kyber_pkey));
 
-  EXPECT_EQ(KYBER512_PUBLIC_KEY_BYTES + KYBER512_PRIVATE_KEY_BYTES, EVP_PKEY_size(kyber_pkey));
-  EXPECT_EQ(8*(KYBER512_PUBLIC_KEY_BYTES + KYBER512_PRIVATE_KEY_BYTES), EVP_PKEY_bits(kyber_pkey));
+  EXPECT_EQ(KYBER512_PUBLIC_KEY_BYTES + KYBER512_SECRET_KEY_BYTES, EVP_PKEY_size(kyber_pkey));
+  EXPECT_EQ(8*(KYBER512_PUBLIC_KEY_BYTES + KYBER512_SECRET_KEY_BYTES), EVP_PKEY_bits(kyber_pkey));
 
   EVP_PKEY_CTX_free(kyber_pkey_ctx);
 }
 
 TEST(Kyber512Test, KEMOperations) {
   // Basic functional test for KYBER512
```

Comment:
Just because now we are here :)
```suggestion
  // Basic functional test for KYBER512.
```